src/cascadia/TerminalSettingsEditor/Appearances.xaml

```diff
@@ -210,6 +212,10 @@
                               IsEnabled="{x:Bind UsingMonospaceFont, Mode=OneWay}" />
                 </StackPanel>
             </local:SettingContainer>
+            <muxc:InfoBar x:Uid="Profile_FontFace_ProportionalFontWarning"
+                          IsOpen="{x:Bind ShowProportionalFontWarning, Mode=OneWay}"

```

Comment:
I'm so disappointed in XAML. We already have `UsingMonospaceFont`, which can be bound via `x:Bind`... but because it's not possible to NOT it using a XAML built-in, we would need to add a `Converter` member _or_ introduce another boolean value that is literally the inverse of `UsingMonospaceFont`. That second one is what you've done here, obviously. :)